### PR TITLE
Fix pkg/project tests

### PR DESCRIPTION
**Pull request by soltysh**

### CODE REVIEW

File: pkg/project/auth/cache.go [hunk 1]
```diff
@@ -190,11 +189,11 @@ func NewAuthorizationCache(reviewer Reviewer, namespaceInterface kcoreclient.Nam
 func (ac *AuthorizationCache) Run(period time.Duration) {
 
 	namespaceReflector := cache.NewReflector(
-		&controller.InternalListWatch{
```
Comment (ncdc) [inline pkg/project/auth/cache.go:193]: I guess I'll come back in later and replace standalone reflectors with shared informers where possible...

**Comment by ncdc** [comment]
LGTM

File: pkg/project/registry/project/proxy/proxy.go [hunk 2]
```diff
@@ -111,8 +111,12 @@ func (s *REST) Watch(ctx apirequest.Context, options *metainternal.ListOptions)
 var _ = rest.Getter(&REST{})
 
 // Get retrieves a Project by name
-func (s *REST) Get(ctx apirequest.Context, name string) (runtime.Object, error) {
-	namespace, err := s.client.Get(name, metav1.GetOptions{})
+func (s *REST) Get(ctx apirequest.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+	opts := metav1.GetOptions{}
+	if options != nil {
```
Comment (sttts) [inline pkg/project/registry/project/proxy/proxy.go:116]: Good catch. @ncdc I remember I saw some unguarded dereferencing of options in some of your PRs, maybe also in mine.

**Comment by ncdc** [comment]
Yeah I probably have at least one. Need to go back and check.

**Pull request merged**